### PR TITLE
LPS-86144 SF

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ElementsDefaultEventHandler.es.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		}
+	}
+
+	copyArticle(itemData) {
+		this._send(itemData.copyArticleURL);
+	}
+
+	delete(itemData) {
+		if (this.trashEnabled || confirm(Liferay.Language.get('are-you-sure-you-want-to-delete-this'))) {
+			this._send(itemData.deleteFolderURL);
```
Comment (jbalsas) [inline modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ElementsDefaultEventHandler.es.js]: Hey @carloslancha, @ealonso, this doesn't seem to we working properly to delete articles. Could you take a look?

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ElementsDefaultEventHandler.es.js (hunk 1 above)
Comment (carloslancha) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ElementsDefaultEventHandler.es.js]: Method is working correctly so maybe is a problem with the passed url. Could you please take a look @ealonso ? Thx.

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**